integration-tests/testutils/common/common.go

```diff
@@ -71,10 +71,11 @@ func (s *SnappySuite) SetUpSuite(c *check.C) {
 			// Always use the installed snappy because we are updating from an old
 			// image, so we should not use the snappy from the branch.
 			output := cli.ExecCommand(c, "sudo", "/usr/bin/snappy", "update")
-			// TODO raise an error if there is no available update.
-			if output != "" {
-				RebootWithMark(c, "setupsuite-update")
-			}
+			expected := "(?ms)" +
+				".*" +
```

Comment:
No really. If I remember correctly, I'm following here @fgimenez' suggestion of making the sprintf just for the line with the parameter.